books/free-programming-books.md

```diff
@@ -1692,6 +1692,7 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Introduction to Computer Science "booksite"](https://introcs.cs.princeton.edu/java/cs/) - by Robert Sedgewick and Kevin Wayne (HTML)
 * [Introduction to Computer science using Java](http://www.programmedlessons.org/Java9/index.html) - Bradley Kjell
 * [Introduction to Programming in Java](http://introcs.cs.princeton.edu/java/home/) - Robert Sedgewick and Kevin Wayne
+* [Introduction to Programming Using Java (5th Edition - revised Nov 2007)] (https://www.iitk.ac.in/esc101/share/downloads/javanotes5.pdf) - David J. Eck (PDF)
 * [Introduction to Programming Using Java](http://math.hws.edu/javanotes/) - David J. Eck
```

Comment:
```suggestion
* [Introduction to Programming Using Java](http://math.hws.edu/javanotes) - David J. Eck (HTML, PDF, ePUB + exercises)
* [Introduction to Programming Using Java (5th Edition - final version, 2010 Jun)](https://math.hws.edu/eck/cs124/javanotes5) - David J. Eck (HTML, PDF, ePUB + exercises)
```

- Solves CI/CD linting errors
- Reference to author site and lastest 5th edition

Reply:
@hrishikeshp7 take in mind resolve this suggested changes and make PR ready to review if you want advance to be merged 😉